doc/man7/EVP_PKEY-DH.pod

```diff
@@ -137,22 +137,21 @@ An B<DH> key can be generated with a named safe prime group by calling:
     EVP_PKEY_free(key);
     EVP_PKEY_CTX_free(pctx);
 
-Legacy B<DH> domain parameters can be generated by calling:
+B<DHX> domain parameters can be generated according to B<FIPS 186-4> by calling:
 
     unsigned int pbits = 2048;
     unsigned int qbits = 256;
-    int gindex = 1;
     OSSL_PARAM params[5];
     EVP_PKEY *param_key = NULL;
     EVP_PKEY_CTX *pctx = NULL;
 
-    pctx = EVP_PKEY_CTX_new_from_name(NULL, "DH", NULL);
+    pctx = EVP_PKEY_CTX_new_from_name(NULL, "DHX", NULL);
     EVP_PKEY_paramgen_init(pctx);
 
     params[0] = OSSL_PARAM_construct_uint("pbits", &pbits);
     params[1] = OSSL_PARAM_construct_uint("qbits", &qbits);
-    params[2] = OSSL_PARAM_construct_int("gindex", &gindex);
```

Comment:
ARRG!!!! Please dont remove things you dont understand.. This needs to be added back in!

Reply:
You have added an unverifiable g by doing this.